src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx

```diff
@@ -110,4 +112,67 @@ export const vaults: InfluenceVaultConfig[] = [
       },
     },
   },
+  {
+    influenceVaultToken: mainnetDeploy.sett_system.vaults['native.graviaura'], // graviAura
+    poolToken: '',
+    vaultToken: mainnetDeploy.tokens['aura'], //Aura
+    roundStart: 1655956800,
+    sources: [mainnetDeploy.sett_system.vaults['native.graviaura'], mainnetDeploy.tokens['badger']],
+    scheduleRoundCutoff: 0,
+    chartInitialSlice: 0,
+    rewardFrequencies: [
+      {
+        name: 'graviAURA, BADGER',
+        value: 'Bi-Weekly',
+      },
+    ],
+    rewardFrequenciesModalConfig: {
+      title: 'Reward Frequency',
+      body: [
+        'graviAURA and BADGER rewards from bribe sales are distributed at the completion of each bi-weekly voting round, after bribes for that round have been sold.',
+      ],
+      points: [
+        ['- 75% of bribe sales (after fees): sold for graviAURA'],
+        ['- 25% of bribe sales (after fees): sold for BADGER'],
```

Comment:
can we check if this is correct? i think it was all emissions although i didn't check closely. if there are no badger token rewards in tree distro then i think its not from bribes

Reply:
it matches the Figma, i'll confirm with ZK and Trit